examples/near_global_ocean_simulation.jl

```diff
@@ -24,15 +24,15 @@ using Dates
 
 # ### Grid configuration 
 #
-# We define a global grid with a horizontal resolution of 1/4 degree and 40 vertical levels.
+# We define a global grid with a horizontal resolution of 1/4 degree and 30 vertical levels. 
```

Comment:
```suggestion
# We define a global grid with a horizontal resolution of 1/4 degree and 30 vertical levels.
```

Reply:
what is the difference here?

Reply:
Deleted the space. But mostly wanted to trigger the ci-examples pipeline ;)

Reply:
I should have commented to avoid creating questions